java/src/jmri/jmrix/ActiveSystemsMenu.java

```diff
@@ -103,10 +101,9 @@ static public void addItems(JMenu m) {
             }
         }
 
-
-        if (jmri.jmrix.easydcc.ActiveFlag.isActive()) {
-            m.add(getMenu("jmri.jmrix.easydcc.EasyDCCMenu"));
-        }
+//        if (jmri.jmrix.easydcc.ActiveFlag.isActive()) { // migrated 2017 4.9.6ish
+//            m.add(getMenu("jmri.jmrix.easydcc.EasyDccMenu"));
```

Comment:
Again, this should be removed, not commented out.

Reply:
done